user-defined-plugin.md

```diff
@@ -250,6 +271,34 @@ If you use `boolean`, you can define a list of values with two values (i.e. true
 
 If the parameter needs default value, you can set it through `defaultValue` attribute.
 
+#### withName
+
+Set `true` if you want to make this argument as "named" parameter. See inputParameter for details.
+
+#### isStringArray
+
+If you want to support `c("a", "b", "c")` R Vector parameter, you an set `isStringArray` as true. In this case, user enters comma(,) separated values in a input field and Data Source Extension Framework convert it to `c("a", "b", "c")` fashion.
```

Comment:
typo: "you an set"

Since `c("a", "b", "c")` is just an example, I would rewrite the beginning of this sentence like following.
If you want to support R character vector parameters like `c("a", "b", "c")`, you can set ...